subprojects/common/report-viewer/src/main/kotlin/com/avito/report/ReportsApiFactory.kt

```diff
@@ -12,7 +12,7 @@ import java.util.concurrent.TimeUnit
 
 object ReportsApiFactory {
 
-    private const val TIMEOUT_SEC = 30L
+    private const val TIMEOUT_SEC = 10L
```

Comment:
I checked the dashboard and see that the longest time is 5s
So I think we don't need to wait 30 seconds